src/lang/en.json

```diff
@@ -502,10 +502,12 @@
       },
       "general": {
         "filingPending": "Filing Pending",
-        "paidButNotCompletedByRegistry": "This FILING is paid, but the filing has not been completed by the Business Registry yet. Some filings may take longer than expected.",
+        "paidButStillBeingProcessed": "This {title} is paid, but is still being processed. Refresh this page or come back later to check on the progress.",
+        "ifThisIssueContinues": "If this issue continues, please contact us.",
```

Comment:
Looks like this:

<img width="910" height="353" alt="image" src="https://github.com/user-attachments/assets/9aa56b6a-7c3f-4960-a0ba-550cc3c34e23" />